bin/kafka-run-class.sh

```diff
@@ -245,7 +245,12 @@ GC_FILE_SUFFIX='-gc.log'
 GC_LOG_FILE_NAME=''
 if [ "x$GC_LOG_ENABLED" = "xtrue" ]; then
   GC_LOG_FILE_NAME=$DAEMON_NAME$GC_FILE_SUFFIX
-  KAFKA_GC_LOG_OPTS="-Xloggc:$LOG_DIR/$GC_LOG_FILE_NAME -verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=100M"
+  JAVA_VERSION=$($JAVA -version 2>&1 | sed -E -n 's/.* version "([^.-]*).*"/\1/p')
+  if [[ "$JAVA_VERSION" -ge "9" ]] ; then
+    KAFKA_GC_LOG_OPTS+="-Xlog:gc*:$LOG_DIR/$GC_LOG_FILE_NAME:time,tags:filecount=10,filesize=102400"
```

Comment:
I found a set of slides that maps old options to new options:

https://www.slideshare.net/PoonamBajaj5/lets-learn-to-talk-to-gc-logs-in-java-9

A few comments:
1. It claims that `-XX:+PrintGCTimeStamps` -> `uptime` decorator. We are using the former, do we want to use the latter?
2. It seems to have a `file=` prefix before the file path, is that optional?
3. We should probably include the `safepoint` decorator (equivalent -XX:+PrintGCApplicationConcurrentTime and -XX:+PrintGCApplicationStoppedTime). Not sure why we don't have them currently.

Thoughts?